pkg/operator/target_config_reconciler.go

```diff
@@ -347,6 +379,76 @@ func (c *TargetConfigReconciler) manageClusterRoleBindingProxy(ctx context.Conte
 	return resourceapply.ApplyClusterRoleBinding(ctx, c.kubeClient.RbacV1(), c.eventRecorder, required)
 }
 
+func (c *TargetConfigReconciler) manageIssuerCR(ctx context.Context, leaderWorkerSetOperator *leaderworkersetapiv1.LeaderWorkerSetOperator) (*unstructured.Unstructured, bool, error) {
+	gvr := schema.GroupVersionResource{
+		Group:    "cert-manager.io",
+		Version:  "v1",
+		Resource: "issuers",
+	}
+
+	required := &unstructured.Unstructured{
+		Object: map[string]interface{}{
+			"apiVersion": "cert-manager.io/v1",
+			"kind":       "Issuer",
+			"metadata": map[string]interface{}{
+				"ownerReferences": []interface{}{
+					map[string]interface{}{
+						"apiVersion": "operator.openshift.io/v1",
+						"kind":       "LeaderWorkerSetOperator",
+						"name":       leaderWorkerSetOperator.Name,
+						"uid":        string(leaderWorkerSetOperator.UID),
+					},
+				},
+				"name":      "selfsigned",
+				"namespace": c.namespace,
+			},
+			"spec": map[string]interface{}{
+				"selfSigned": map[string]interface{}{},
+			},
+		},
+	}
+
+	return resourceapply.ApplyUnstructuredResourceImproved(ctx, c.dynamicClient, c.eventRecorder, required, c.resourceCache, gvr, nil, nil)
+}
+
+func (c *TargetConfigReconciler) manageCertificateWebhookCR(ctx context.Context, leaderWorkerSetOperator *leaderworkersetapiv1.LeaderWorkerSetOperator) (*unstructured.Unstructured, bool, error) {
+	gvr := schema.GroupVersionResource{
+		Group:    "cert-manager.io",
+		Version:  "v1",
+		Resource: "certificates",
+	}
+
+	required := &unstructured.Unstructured{
+		Object: map[string]interface{}{
+			"apiVersion": "cert-manager.io/v1",
+			"kind":       "Certificate",
+			"metadata": map[string]interface{}{
+				"ownerReferences": []interface{}{
+					map[string]interface{}{
+						"apiVersion": "operator.openshift.io/v1",
+						"kind":       "LeaderWorkerSetOperator",
+						"name":       leaderWorkerSetOperator.Name,
+						"uid":        string(leaderWorkerSetOperator.UID),
+					},
+				},
+				"name":      "webhook-cert",
+				"namespace": c.namespace,
+			},
+			"spec": map[string]interface{}{
+				"secretName": "webhook-server-cert",
+				"dnsNames": []interface{}{
+					"lws-webhook-service.openshift-lws-operator.svc",
```

Comment:
This probably should be using the operator namespace?

Reply:
You are absolutely right. Thank you. Updated.